tests/unit/test_fvm_multi.cpp

```diff
@@ -191,18 +194,47 @@ TEST(fvm_multi, stimulus)
     fvm_cell fvcell;
     fvcell.initialize(singleton_view(cell), detectors, targets, probes);
 
-    auto& stim = fvcell.stimuli();
-    EXPECT_EQ(stim.size(), 2u);
+    auto ref = fvcell.find_mechanism("stimulus");
+    if (ref) {
+        auto& stims = ref.get();
```

Comment:
Simpler just to fail early if `!ref`, i.e.
```
ASSERT_TRUE(ref) << "no stimuli retrieved ...";
```
and then no need for `if (ref) { ...`